models/olids/base/base_olids_medication_order.sql

```
@@ -59,31 +59,36 @@ SELECT
     src.lds_is_deleted,
-    src.lds_start_date_time,
+    src.lds_start_datetime,
     src.lds_lakehouse_date_processed,
     src.lds_lakehouse_datetime_updated
 FROM {{ source('olids_common', 'MEDICATION_ORDER') }} src
 INNER JOIN {{ ref('base_olids_patient') }} patients
     ON src.patient_id = patients.id
 INNER JOIN {{ ref('int_wnl_practices') }} wnl_practices
-    ON src.record_owner_organisation_code = wnl_practices.practice_code
+    ON src.publisher_organisation_code = wnl_practices.practice_code
 LEFT JOIN {{ source('olids_common', 'MEDICATION_STATEMENT') }} ms
```

SQLFluff (4.2.1)

[error] Implicit/explicit aliasing of table.

(AL01)